src/clj/game/cards/assets.clj

```diff
@@ -38,16 +38,15 @@
   Trashes itself when out of counters"
   [counters per-turn]
   (let [ability {:msg (str "gain " per-turn " [Credits]")
-                 :counter-cost [:credit per-turn]
                  :once :per-turn
                  :req (req (:corp-phase-12 @state))
                  :label (str "Gain " per-turn " [Credits] (start of turn)")
-                 :effect (req (gain-credits state :corp per-turn)
-                              (when (zero? (get-counters card :credit))
-                                (trash state :corp card)))}]
+                 :effect (effect (gain-credits (min per-turn (get-counters card :credit)))
```

Comment:
Future-proofing. I love it :+1:

Reply:
Yeah, I’ve heard whispers of an Uprising card that places credits on cards, and technically these abilities aren’t costs, they’re effects, so you always do as much as possible.